File: hsg9-tinh/2023-2024-haiduong/tongchuso/tongchuso.cpp
#include <iostream>

#define input_file "tongchuso.inp"
#define output_file "tongchuso.out"

using namespace std;

int n;
int Sum = 0;


void input()
{
    ios_base::sync_with_stdio(false);
    cin.tie(NULL);

    freopen(input_file, "r", stdin);

    cin >> n;
}


void process()
{
    while (n > 0)
    {
        Sum += n % 10;
        n /= 10;
    }
}


void output()
{
    freopen(output_file, "w", stdout);
    
    cout << Sum;
}


int main()
{
    input();
    process();
    output();

    return 0;
}

File: hsg9-tinh/2023-2024-haiduong/tongchuso/tongchuso.py
import os
import sys

input_file = os.path.join(sys.path[0], 'tongchuso.inp')
output_file = os.path.join(sys.path[0], 'tongchuso.out')

n = 0
Sum = 0


def input():
    global n

    with open(input_file, 'r') as f:
        n = int(f.readline())


def process():
    global n, Sum

    while n > 0:
        Sum += n % 10
        n = n // 10


def output():
    global Sum

    with open(output_file, 'w') as f:
        f.write(str(Sum))


if __name__ == '__main__':
    input()
    process()
    output()
